### PR TITLE
Bumping a few deps

### PR DESCRIPTION
**Pull request by oldergod**
I cannot upgrade protobuf because everything breaks and I don't know how to fix it right now.

### CODE REVIEW

File: wire-library/dependencies.gradle [hunk 1]
```diff
@@ -2,19 +2,18 @@ ext.versions = [
...
-  'javapoet': '1.11.1',
+  'javapoet': '1.13.0',
   'kotlinpoet': '1.6.0',
   'jsr305': '3.0.2',
   'kotlin': '1.3.71',
-  'okio': '2.5.0',
+  'okio': '2.8.0',
   'okhttp': '4.2.2',
   'moshi': '1.9.2',
-  'protobuf': '3.11.4',
```
Comment (oldergod) [inline wire-library/dependencies.gradle:15]: this was never used :(

**Comment by oldergod** [comment]
Not sure why okio isn't find on multiplatform in CI. It works locally.

File: wire-library/dependencies.gradle [hunk 2]
```diff
@@ -2,19 +2,18 @@ ext.versions = [
...
   'guava': '20.0',
-  'javapoet': '1.11.1',
+  'javapoet': '1.13.0',
   'kotlinpoet': '1.6.0',
   'jsr305': '3.0.2',
   'kotlin': '1.3.71',
-  'okio': '2.5.0',
+  'okio': '2.8.0',
   'okhttp': '4.2.2',
   'moshi': '1.9.2',
```
Comment (swankjesse) [inline wire-library/dependencies.gradle:14]: 1.10.0 now!

Comment (oldergod) [reply wire-library/dependencies.gradle]: fixed !

**Pull request merged**